docs/_news/news07.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: news
+title: New for Codewind 0.7.0
+description: New for Codewind 0.7.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news07
+---
+
+## Codewind 0.7.0
+// DOUBLE CHECK -- Monday 16 December 2019
+
+#### :sparkles: Remote Deploy with Codewind: Tech Preview :sparkles:
+
+You can now develop your code locally and securetly build and deploy your apps in the cloud! Why is this cool?
+- Free up local resources. :clap:
+- It's secure! We use [Keycloak](https://keycloak.org/) to secure the connection between your local editor and remote cloud deployment.
+
+// Curious? [Try it out!](link to docs) or [learn more](Link to our tech preview blog)
+
+#### New Features and Highlights for 0.7.0
```

Comment:
In the docs, we've been using lower case for everything except for the first word of the title and names of things. I'm OK with leaving the capitalization like this for the news--as long as we keep the news pages titles consistent with themselves. :)

Reply:
Thanks for the advice!